mxnet-bot/LabelBotAddLabels/README.md

```diff
@@ -0,0 +1,70 @@
+# label bot
+This bot serves to help non-committers add labels to GitHub issues.
+
+"Hi @mxnet-label-bot, please add labels: [operator, feature request]"
+
+## Setup
+
+
+#### 1. Store a secret
```

Comment:
Can we add a terraform template to store this secret? So we can automate this step?


https://www.terraform.io/docs/providers/aws/r/secretsmanager_secret.html

Reply:
Hi Pedro, the AWS secret step was originally automated in her script. However we think it may not be fully secured if we place the secret during the deployment of the package. Upon discussion with @marcoabreu , secret settings should go individually from this design as it won't be changed all the time when we deploy. Apart from that, the secret removal on AWS Secret Manager is not happened immediately. It will be expired and eventually removed after a time period.

Reply:
Hi Pedro, at first all steps were automated using boto3. But in that case secrets would be hard coded. After thinking up with @marcoabreu , I used the serverless framework to deploy. And decoupled this step as manual steps.

Reply:
Secrets can be in a terraform var. You can still use serverless. It's for the secrets part to avoid manual console work which is anti pattern

Reply:
Anyway this is not a big deal nor blocker

Reply:
In Terraform you can define variables that have to be input during runtime or configured in a file. A decoupled process that accepts the secret as a runtime parameter and creates the secret on your behalf would then be the output. This means that this Terraform script would only have to be once and would be independent of the deployment of the bot itself.
It should be no big deal to set it up, considering you would only create a single Terraform resource and nothing else. Try giving it a shot, it's quite convenient to use.